analytics/manager.go

```diff
@@ -260,18 +260,13 @@ func (m *manager) startUploader(errHandler util.ErrorFunc) {
 }
 
 func (m *manager) upload(tenant, file string, numRecs int) {
-	promLabels := prometheus.Labels{"file": file}
 	prometheusInstrumentedWork := func(ctx context.Context) error {
 		err := m.uploader.workFunc(tenant, file)(ctx)
 		if err == nil {
-			prometheusRecordsByFile.Delete(promLabels)
-			org, env, err := getOrgAndEnvFromTenant(tenant)
-			if err != nil {
-				log.Warnf("unable to set uploaded status count: %s", err)
-			} else {
-				countLabels := prometheus.Labels{"org": org, "env": env, "status": "uploaded"}
-				prometheusRecordsCount.With(countLabels).Add(float64(numRecs))
-			}
+			org, env, _ := getOrgAndEnvFromTenant(tenant)
```

Comment:
Why are we not checking the error from this method any more?

Reply:
There's no point. 1. It should never fail anyway, but also: 2. If it fails, this way we can still track in prometheus.